src/book/mod.rs

```diff
@@ -379,7 +383,7 @@ impl MDBook {
 
                     println!("[*]: Testing file: {:?}", path);
 
-                    let output_result = Command::new("rustdoc").arg(&path).arg("--test").output();
+                    let output_result = Command::new("rustdoc").arg(&path).arg("--test").args(&library_args).output();
```

Comment:
Yes. With no/empty arguments, it simply return `self`.

https://github.com/rust-lang/rust/blob/master/src/libstd/process.rs#L403